docs/index.rst

```diff
@@ -13,6 +11,38 @@
    <a class="github-button" href="https://github.com/run-house/runhouse/fork" data-icon="octicon-repo-forked" data-size="large" aria-label="Fork run-house/runhouse on GitHub">Fork</a>
    </p>
 
+Why Runhouse?
```

Comment:
I think we can drop this or move it below the intro sentence. Page layout look kind of odd when you go straight from H1 to H2 without any intro paragraph. Technically the sentence following is more "what" than "why anyway.

Reply:
<img width="1027" alt="image" src="https://github.com/user-attachments/assets/44d1f1a2-01eb-43e6-9870-4d5c36b318d7">
I am not sure but I thought it felt much weirder to launch into normal text without some kind of subheader?

Reply:
Hmm I don't super love it either way somehow lol

Reply:
It's pretty standard to go H1 -> content -> H2 -> more content

https://modal.com/docs/guide
https://python.langchain.com/docs/introduction/
https://docs.baseten.co/welcome
https://reactnative.dev/docs/getting-started
https://nextjs.org/docs (minimal but still)

Reply:
Went down a rabbit hole questioning whether or not I've been thinking of this wrong, but feeling better with Google recommending not to have headers without text - https://developers.google.com/style/headings#heading-and-title-format